history/views.py

```diff
@@ -564,7 +567,8 @@ def profit_view(request):
     # get data
     data = {}
     for t in Trade.objects.filter(symbol=symbol, status='fill').order_by('-created_on').all():
-        date = datetime.datetime.strftime(t.created_on-datetime.timedelta(hours=7), '%Y-%m-%d')
+        mst_dt = timezone.localtime(t.created_on, pytz.timezone('MST'))
+        date = datetime.datetime.strftime(mst_dt, '%Y-%m-%d')
```

Comment:
MST was just what i used because thats where i live.  might be worth making this a `settings.DISPLAY_TIMEONE` for others to configure.